easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.2-CUDA-9.0.176.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.0.2'
+cuda_version = '9.0.176'
+
+cuda_version_major_minor = '.'.join(cuda_version.split('.')[:2])
+
+versionsuffix = '-CUDA-%s' % cuda_version
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+sources = [{
+    'filename': '%%(namelower)s-%s-linux-x64-v%%(version)s.tgz' % cuda_version_major_minor,
+    'download_filename': '%%(namelower)s-%s-linux-x64-v%%(version_major)s.tgz' % cuda_version_major_minor,
```

Comment:
@smoors Ah, this isn't going to work, since EasyBuild is not downloading the tarball itself, it must be provided...

If you want to go forward with the modified filename, just include a clear comment like this instead:

```python
# Nvidia developer registration required.
# download link: https://developer.nvidia.com/rdp/cudnn-download
# download as cudnn-9.0-linux-x64-v7.tgz, rename to cudnn-9.0-linux-x64-v7.0.2.tgz
sources = ['%%(namelower)s-%s-linux-x64-v%%(version)s.tgz' % cuda_version_major_minor]

Reply:
ok, thx!